config/index.js

```diff
@@ -104,6 +104,8 @@ module.exports = {
   },
   ENABLE_COMPONENTS_LIBRARY:
     IS_DEV || /true/i.test(process.env.ENABLE_COMPONENTS_LIBRARY),
+  ENABLE_DEVELOPMENT_TOOLS:
```

Comment:
Is there any way we could ensure that this could never be set on production. (ie. currently, a naughty person with access could add the env var to the k8s config and simply apply it)

Reply:
Any environment variable is susceptible to that sort of manipulation.

Best option I can think of involves denying this path in the ingress configuration.

Only other two options I can think off are either not including this module in our production builds, which is not a great solution, because then what is staging actually representing. Alternatively, if this was a separate microservice stitched together at the Ingress layer we wouldn't need to worry about this - but that's effectively the same as denying it at that later anyway.

Reply:
I did also toy with this but the reason I created the env was to actually potentially allow it on some "production-like" environments like Heroku or Staging.

I think some of the tools we may add to this will be useful in those environments, like pre-filling a PER. 

However, you're right that this does potentially open up security concerns if things like permissions can be manipulated.

Maybe it is safer at this stage that it can **only** be done in development environments and we remove support for the environment variable altogether.

@merlinc's suggestion is also something we thought about with the original development only middleware. We could potentially add a npm task that cleans up after a deploy to remove such files?